apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.html

```diff
@@ -0,0 +1,36 @@
+<fd-card cardType="analytical">
+    <fd-card-header
+        >Describes the type of
+        <fd-card-title>Projected revenue</fd-card-title>
+        <fd-card-subtitle>Values include taxes | USD</fd-card-subtitle>
```

Comment:
This is same case as title component. It would be great to make it as directive

Reply:
Changed it to directive